### ISSUE TITLE
No cookies received from the server

### ISSUE THREAD

**Issue by sagarmsraozil** [microcipcip/cookie-universal#132]
I want the cookie to be set in my frontend from server. But in my postman, when i run the login api. It does not set the cookie. Instead gives the message like "No cookies received from the server." 

cookies.set('accessToken', userToken, {
              path: '/',
              maxAge: 1000 * 60 * 60 * 24 * 1,
              domain: '.localhost',
              httpOnly: true,
            });
            
Any help is appreciated.

**Comment by microcipcip** [comment]
So you're using the cookie inside postman?

**Comment by sagarmsraozil** [comment]
Yes, i check my apis from cookie before merging it. Is it possible?

**Comment by microcipcip** [comment]
Are you sure that the `domain` is correct? Is `.localhost` the postman domain?

**Comment by sagarmsraozil** [comment]
so, i need to specify the domain of postman? as the server is running in localhost

**Comment by microcipcip** [comment]
Can you try with `localhost` instead of `.localhost`? The domain has to be exact or it will not send the cookie.
